modules/402-ingress-nginx/templates/kruise/manager.yaml

```diff
@@ -0,0 +1,149 @@
+{{- define "manager_resources" }}
+cpu: 100m
+memory: 128Mi
+{{- end }}
+
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: kruise-controller-manager
+  namespace: d8-ingress-nginx
+  {{- include "helm_lib_module_labels" (list . (dict "app" "kruise" "control-plane" "controller-manager" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: kruise-controller-manager
+  updatePolicy:
+    updateMode: "Auto"
+  resourcePolicy:
+    containerPolicies:
+      - containerName: kruise
+        minAllowed:
+          {{- include "manager_resources" . | nindent 10 }}
+        maxAllowed:
+          cpu: 300m
+          memory: 512Mi
+{{- end }}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: kruise-controller-manager
+  namespace: d8-ingress-nginx
+  {{- include "helm_lib_module_labels" (list . (dict "app" "kruise")) | nindent 2 }}
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app: kruise
+---
+# Source: kruise/templates/manager.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kruise-controller-manager
+  namespace: d8-ingress-nginx
+  {{- include "helm_lib_module_labels" (list . (dict "app" "kruise" "control-plane" "controller-manager")) | nindent 2 }}
+spec:
+  {{- include "helm_lib_deployment_strategy_and_replicas_for_ha" . | nindent 2 }}
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app: kruise
+  minReadySeconds: 3
```

Comment:
🤔

Reply:
? It's a default value from official kruise chart. Have no idea, why we should change it